### ISSUE TITLE
feature request: include SweepFinder's CLR statistic

### ISSUE THREAD

**Issue by mufernando** [kr-colab/pg_gpu#98]
The software is so slow and, more importantly, it does not take accessibility in account! Many of the outliers positions I found with real data were due to missingness skewing the SFS.

**Comment by nspope** [comment]
I think this is non-trivial because it requires optimizing free parameters in a (phenomenological) model--- which *is* possible to accelerate on a GPU via parallelized Newton-Raphson (e.g. with no line search) but is pretty fundamentally different from the rest of the statistics. Any serial searching (bisection, etc) during optimization wouldn't be a good fit for the GPU.
